packages/flutter_tools/lib/src/template.dart

```diff
@@ -84,13 +85,14 @@ class Template {
         if (overwriteExisting) {
           finalDestinationFile.delete(recursive: true);
           printStatus('$relativePathForLogging exists. Overwriting.');
+          createdFileCount++;
         } else {
           // The file exists but we cannot overwrite it, move on.
           printStatus('$relativePathForLogging exists. Skipping.');
           return;
         }
       } else {
-        printStatus('$relativePathForLogging created.');
```

Comment:
I admit, this used to be very useful to me to troubleshoot the `flutter create` command.